packages/pds/src/api/com/atproto/repo/uploadBlob.ts

```diff
@@ -6,7 +6,9 @@ import { BlobMetadata } from '../../../../actor-store/blob/transactor'
 
 export default function (server: Server, ctx: AppContext) {
   server.com.atproto.repo.uploadBlob({
-    auth: ctx.authVerifier.accessCheckTakedown,
+    auth: ctx.authVerifier.accessStandard({
+      checkTakedown: true,
+    }),
```

Comment:
No biggie either way, but I imagine we could do a deactivated step too since this is part of a write flow.

Reply:
Yeah I originally did that, but uploadBlob is used in the account migration flow (during which the account is deactivated). Since `uploadBlob` doesn't actually produce firehose events, I figured it was okay to let it through

May be worth revisiting if we want an account that is in the process of importing their repo to actually be "deactivated"